src/iosrtcPlugin.swift

```diff
@@ -331,6 +331,44 @@ class iosrtcPlugin : CDVPlugin {
 		}
 	}
 
+    func RTCPeerConnection_getStats(command: CDVInvokedUrlCommand) {
+        NSLog("iosrtcPlugin#RTCPeerConnection_getStats()")
+
+        let pcId = command.argumentAtIndex(0) as! Int
+        let pluginRTCPeerConnection = self.pluginRTCPeerConnections[pcId]
+
+        if pluginRTCPeerConnection == nil {
+            NSLog("iosrtcPlugin#RTCPeerConnection_getStats() | ERROR: pluginRTCPeerConnection with pcId=\(pcId) does not exist")
```

Comment:
I agree with your comments. Again, I literally copied this approach from similar functions in this file.